Makefile

```diff
@@ -0,0 +1,9 @@
+DOCKER_RUN = docker-compose run --rm php
+
+vendor:
+	${DOCKER_RUN} composer install --no-interaction
+
+.PHONY: test
```

Comment:
Coding style comment: adding all phony at the beginning of thé document is easier to read